guide/ops/https.md

```diff
@@ -0,0 +1,53 @@
+---
+title: HTTPS Configuration
+layout: website-normal
+---
+
+## HTTPS Configuration
+
+### Getting the Certificate
+To enable HTTPS web access, you will need a server certificate in a java keystore. To create a self-signed certificate,
+and add it to a keystore, `keystore.jks`, you can use the following command:
+
+{% highlight bash %}
+% keytool -genkey -keyalg RSA -alias brooklyn \
+          -keystore <path-to-keystore-directory>/keystore.jks -storepass "mypassword" \
+          -validity 365 -keysize 2048 -keypass "password"
+{% endhighlight %}
+
+Of course, the passwords above should be changed.  Omit those arguments above for the tool to prompt you for the values.
```

Comment:
Perhaps change "Of course, the passwords above should be changed" to "The passwords above should be changed to something [more secure](https://en.wikipedia.org/wiki/Password_strength)"